addons/binding/org.openhab.binding.opensprinkler/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+    <classpathentry exported="true" kind="lib" path="lib/pi4j-core.jar"/>

```

Comment:
Could you add an about.html similar to https://github.com/openhab/openhab2/blob/master/bundles/io/org.openhab.io.jetty.certificate/about.html, so that you can clearly name this 3rd party lib and provide its license terms?